### PR TITLE
Docs migrate zuora latest dh

### PR DESCRIPTION
**Pull request by dhirsch1**
PR to merge new template files for Zuora.

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -285,4 +285,8 @@
  ** xref:xml/xml-schema-validation.adoc[Validate Documents against an XSD Schema with the XML Module]
  ** xref:xml/xml-reference.adoc[XML Module Reference]
 * xref:zuora/zuora-connector.adoc[Zuora Connector]
- ** xref:zuora/zuora-connector-reference.adoc[Zuora Connector Reference]
```
Comment (mcherem) [inline modules/ROOT/nav.adoc:288]: I guess you got your links to work?

File: modules/ROOT/pages/zuora/zuora-connector-design-center.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+== Configure the Input Source Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*, and click *Create*.
+. Exit from *Let's get started* by clicking *Go straight to canvas*.
+. Click on the Trigger card.
+. Search for and select *HTTP Listener* or *Scheduler*.
+. Complete the fields with the values you want to configure for your trigger. For more information on field values, see the xref:zuora/zuora-connector-reference.adoc[Zuora Connector Reference].
```
Comment (mcherem) [inline modules/ROOT/pages/zuora/zuora-connector-design-center.adoc]: The correct title of the Mule 4 Connector Reference is: 
Zuora Connector Reference - Mule 4

File: modules/ROOT/pages/zuora/zuora-connector-design-center.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+== Configure the Zuora Target Component
+
+To add a Zuora connector as a component:
+
+. Click “+” next to the trigger card.
```
Comment (mcherem) [inline modules/ROOT/pages/zuora/zuora-connector-design-center.adoc]: The plus should be in bold.

File: modules/ROOT/pages/zuora/zuora-connector-examples.adoc [hunk 4]
```diff
@@ -0,0 +1,68 @@
+= Zuora Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Example: Retrieve Account Details for an Account
```
Comment (mcherem) [inline modules/ROOT/pages/zuora/zuora-connector-examples.adoc]: Any explanatory text after this heading and before the next?

File: modules/ROOT/pages/zuora/zuora-connector-examples.adoc [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+== Studio Flows
+
+* Listener (HTTP) - Accepts data from HTTP requests.
+* Zuora API Connector - Connects with Zuora and executes a query to retrieve the account details for the specified account ID.
+* Transform message - Outputs the results of the `Get account` operation in JSON format.
```
Comment (mcherem) [inline modules/ROOT/pages/zuora/zuora-connector-examples.adoc:15]: Backticks on JSON?

File: modules/ROOT/pages/zuora/zuora-connector-examples.adoc [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
++
+image::zuora/zuora-studio-use-case.png[studio_use_case]
```
Comment (mcherem) [inline modules/ROOT/pages/zuora/zuora-connector-examples.adoc:18]: Would question how necessary this image is here. It does not seem to add anything beyond the text description.

File: modules/ROOT/pages/zuora/zuora-connector-studio.adoc [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
++
+** *basePath*: Base path of the REST API for your instance. Usually this is the `/rest` path.
+** *username*: Zuora username.
+** *password*: Zuora password.
+** *host*: Path to the Zuora API, for example: `apisandbox-api.zuora.com`.
+** *port*: Port where the Zuora API listens for requests.
+** *protocol*: Protocol to use for accessing the Zuora API.
++
+
+image::zuora/zuora-basic-auth-studio.png[Zuora Connector Configuration in Studio 7]
```
Comment (mcherem) [inline modules/ROOT/pages/zuora/zuora-connector-studio.adoc:38]: Again, just not so sure how helpful this image is here. I was told by George to remove all of them in the FD and Studio templates.

File: modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+following actions to get access to the connector:
+
+. Add the namespace for the connector to your XML flow.
+. Add dependency information to your Maven `pom.xml` file.
+
+In the Studio visual editor, when you drag a connector from the palette to the canvas, Studio automatically populates the XML code with the connector namespace and schema location.
+
+== Add a Namespace for the Connector
+
+If you manually code the Mule app using Studio’s XML editor or another text editor, paste the following code inside the <mule> tag of the header of your configuration XML.
```
Comment (mcherem) [inline modules/ROOT/pages/zuora/zuora-connector-xml-maven.adoc]: does the <mule> tag  need to be tagged for formatting?

**Review by mcherem — APPROVED** [review]
Peer review.

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (dhirsch1) [reply modules/ROOT/nav.adoc:288]: Yes, but I'll test again to make sure.

File: modules/ROOT/pages/zuora/zuora-connector-design-center.adoc (hunk 2 above)
Comment (dhirsch1) [reply modules/ROOT/pages/zuora/zuora-connector-design-center.adoc]: I am not sure we need to say Mule 4. I haven't been doing this but you are probably correct so I will change it.

**Pull request merged**